### PR TITLE
44 accordion header design

### PR DESCRIPTION
**Pull request by willbreitkreutz**
I kind of like this, just added a 1px border and it breaks up the accordion headers. This PR depends on #66 so is set to draft.

### CODE REVIEW

**Comment by willbreitkreutz** [comment]
image

**Comment by jbkolze** [comment]
So, in the course of reviewing this I came across another problem on our favorite topic: conflicts between groundwork and user tailwind versions!  The original comment was made in reference to the Accordion implementation in lrl-wm-web, and I just realized that the base gw-bg-gray-50 isn't being applied to Accordion headers in lrl-wm-web... so they're blending into the surrounding content even more so than they should be.

Long story short, it looks like the gw-bg-gray-50 from groundwork is being overwritten by a rogue "transparent" on buttons from the base tailwind styling:
image

I can fix it by importing the groundwork style.css **after** the local index.css, but of course that causes some other local stuff to break.

Haven't really dug in enough to nail down the exact cause yet.  Seems like it might be a tricky one.

**Comment by jbkolze** [comment]
So, in the course of reviewing this I came across another problem on our favorite topic: conflicts between groundwork and user tailwind versions!  The original comment was made in reference to the Accordion implementation in lrl-wm-web, and I just realized that the base gw-bg-gray-50 isn't being applied to Accordion headers in lrl-wm-web... so they're blending into the surrounding content even more so than they should be.

Long story short, it looks like the gw-bg-gray-50 from groundwork is being overwritten by a rogue "transparent" on buttons from the base tailwind styling:
image

I can fix it by importing the groundwork style.css **after** the local index.css, but of course that causes some other local stuff to break.

Haven't really dug in enough to nail down the exact cause yet.  Seems like it might be a tricky one.

**Comment by willbreitkreutz** [comment]
@jbkolze just want to make sure that it's using the gwMerge (from the other PR that hasn't been merged yet) in the version you're testing

**Comment by jbkolze** [comment]
@willbreitkreutz Yeah, tested specifically with this groundwork branch linked to my repo, so the gwMerge was incorporated.

**Comment by willbreitkreutz** [comment]
Ok, good, or maybe not good... will dig into it

**Comment by willbreitkreutz** [comment]
Closing because I have another set of changes based on a newer point in time that makes more sense.
